redwood-widget/src/iosMain/kotlin/app/cash/redwood/widget/UIViewChildren.kt

```diff
@@ -67,7 +67,10 @@ public class UIViewChildren(
   }
 
   private fun invalidate() {
-    parent.setNeedsLayout()
-    parent.invalidateIntrinsicContentSize()
+    parent.superview?.let {
+      it.setNeedsLayout()
+    } ?: run {
+      parent.setNeedsLayout()
+    }
```

Comment:
Dying for `if let` rather than this (very common) Kotlin pattern.

How about
```suggestion
    (parent.superview ?: parent).setNeedsLayout()
```
which hopefully is a bit more simple to parse.

The rules around iOS layout still elude me...

Reply:
me too. this is a hack that I'm hoping to find another way around